test/functional/wallet_bumpfee.py

```diff
@@ -73,6 +73,10 @@ def run_test(self):
         test_unconfirmed_not_spendable(rbf_node, rbf_node_address)
         test_bumpfee_metadata(rbf_node, dest_address)
         test_locked_wallet_fails(rbf_node, dest_address)
+        test_change_script_match(rbf_node, dest_address)
+        # These tests wipe out a number of utxos that are expected in other tests
```

Comment:
This is a shame. At the moment I think the subtests can be run in any order. It'd be nice to keep that property (so, for example, anyone adding new subtests doesn't need to worry about state carried over from previous tests). Is it possible to rewrite your subtests in such a way to only use the existing outputs and not generate new blocks? For the `test_no_more_inputs_fails()` test you could lock all outputs and then unlock them, for example.